src/SASViyaApiClient.ts

```diff
@@ -589,7 +589,7 @@ export class SASViyaApiClient {
       if (expectWebout) {
         resultLink = `/compute/sessions/${executionSessionId}/filerefs/_webout/content`
       } else {
-        return currentJob
+        return { result: currentJob, log }
```

Comment:
We're returning the `currentJob` object as `job` on line 584:
https://github.com/sasjs/adapter/pull/169/files#diff-72e1fde4c9683400219b3188eac4c2446ec1cfb314e073ef4f5640e0784c207fR584
So we should probably do the same here.

Reply:
Good point